frontend/luasettings.lua

```diff
@@ -60,6 +60,10 @@ function LuaSettings:flipTrue(key)
     end
 end
 
+function LuaSettings:replace(table)
```

Comment:
Right, I was thinking passing in an extra argument would indicate resetting the data with a new one.

I guess a better name would be `setData`. If you still think `replace` is better, I am ok with it too.